### PR TITLE
Revert localized name (ko)

### PR DESCRIPTION
**Pull request by alexkoala**
Reverts localized programming language name (ex. `파이썬`, `루비`) and updated some files to lastest website version.

### CODE REVIEW

File: ko/libraries/index.md [hunk 1]
```diff
@@ -20,21 +20,21 @@ lang: ko
...
 
-루비의 라이브러리는 보동 [**RubyGems.org**][1]에서 젬으로 제공됩니다. 직접
+Ruby의 라이브러리는 보동 [**RubyGems.org**][1]에서 gem으로 제공됩니다. 직접
 웹사이트에서 둘러보거나 `gem` 명령어로 이용할 수 있습니다.
 
 `gem search -r` 명령어를 사용해 RubyGems의 저장소를 조사할 수 있습니다. 예를 들어
-`gem search -r rails`는 레일즈와 관련된 젬들의 목록을 돌려줍니다.
-`--local` (`-l`) 옵션을 사용하면 설치된 젬들에 대해 로컬 검색할 수 있습니다. 젬을
-설치하려면 `gem install [gem]`을 사용합니다. 설치된 젬을 보려면 `gem
+`gem search -r rails`는 레일즈와 관련된 gem들의 목록을 돌려줍니다.
```
Comment (yous) [inline ko/libraries/index.md:27]:
```suggestion
`gem search -r rails`는 Rails와 관련된 gem의 목록을 돌려줍니다.
```

File: ko/libraries/index.md [hunk 2]
```diff
@@ -20,21 +20,21 @@ lang: ko
...
+`--local` (`-l`) 옵션을 사용하면 설치된 gem들에 대해 로컬 검색할 수 있습니다. gem을
```
Comment (yous) [inline ko/libraries/index.md:28]:
```suggestion
`--local` (`-l`) 옵션을 사용하면 설치된 gem에 대해 로컬 검색할 수 있습니다. gem을
```

File: ko/libraries/index.md [hunk 3]
```diff
@@ -59,12 +59,12 @@ html-sample (1.0, 1.1)
...
 `--remote` / `-r` 플래그는 공식 RubyGems.org 저장소에서 검색하려 한다는
-의미입니다. (기본 동작) `--local` / `-l` 플래그로는 설치된 젬에 대해
+의미입니다. (기본 동작) `--local` / `-l` 플래그로는 설치된 gem에 대해
 검색하게 됩니다.
 
-#### 젬 설치하기
+#### gem 설치하기
 
-젬을 설치하려면 **install** 명령을 사용하면 됩니다. 예를 들어 레일즈를
+gem을 설치하려면 **install** 명령을 사용하면 됩니다. 예를 들어 레일즈를
```
Comment (yous) [inline ko/libraries/index.md:67]:
```suggestion
gem을 설치하려면 **install** 명령을 사용하면 됩니다. 예를 들어 Rails를
```

File: ko/security/index.md [hunk 4]
```diff
@@ -1,10 +1,10 @@
 ---
 layout: page
-title: "보안이슈"
+title: "보안"
 lang: ko
 ---
 
-루비와 관련한 보안이슈에 대해 정보를 공유하는 곳입니다.
+와 관련한 보안 이슈에 대해 정보를 공유하는 곳입니다.
```
Comment (yous) [inline ko/security/index.md:7]:
```suggestion
Ruby와 관련한 보안 이슈에 대해 정보를 공유하는 곳입니다.
```

**Comment by yous** [comment]
Seems like Korean version of Python doc site uses '파이썬' too: https://docs.python.org/ko/3/

But I'm okay with both 'Ruby' and '루비'.

File: ko/about/index.md [hunk 5]
```diff
@@ -1,62 +1,62 @@
...
-이런 성장은 루비로 만들어진 대중적인 소프트웨어, 특히 [루비 온 레일즈][ror] 웹 프레임워크의 공입니다.
+Ruby는 성장세와 인기 면에서 [TIOBE 인덱스][tiobe] 같은 세계적인 랭킹의 10위권 안에 드는 프로그래밍 언어입니다.
+이런 성장은 Ruby로 만들어진 대중적인 소프트웨어, 특히 [Ruby 온 레일즈][ror] 웹 프레임워크의 공입니다.
 
-루비는 [완전히 무료]({{ site.license.url }})입니다. 사용뿐만 아니라 복사, 수정, 배포까지도 무료입니다.
+Ruby는 [완전히 무료]({{ site.license.url }})입니다. 사용뿐만 아니라 복사, 수정, 배포까지도 무료입니다.
 
 ### 모든 것은 객체
 
 처음에 마츠는 이상적인 문법을 가진 언어를 찾고 있었습니다. 그 때를 회고하며 그는 “나는 펄보다 강력하고 파이썬보다는 객체지향적인
```
[the remark this answers is not in the document]
Comment (alexkoala) [reply ko/about/index.md:39]: Perl doesn't have website translation. In a case of Python, only Korean document uses localized name `파이썬`.

File: ko/about/index.md [hunk 6]
```diff
@@ -1,62 +1,62 @@
...
+Ruby가 왜 인기 있을까요? Ruby의 팬들은 아름답고 기교가 뛰어난 언어이기 때문이라고 말합니다. 하지만 이 말로는 부족하지요. 어떤
+점이 Ruby를 이처럼 매력적인 언어로 만드는 것일까요?
 {: .summary}
 
-### 루비를 만든 마츠의 목표
+### Ruby를 만든 마츠의 목표
 
-루비는 균형잡힌 언어입니다. 루비를 만든 [유키히로 마츠모토][matz](일명 마츠)는 그가 좋아하는 언어인 펄, 스몰토크, 아이펠,
+Ruby는 균형잡힌 언어입니다. Ruby를 만든 [유키히로 마츠모토][matz](일명 마츠)는 그가 좋아하는 언어인 펄, 스몰토크, 아이펠,
 에이다, 리스프를 섞어서 명령형 프로그래밍언어이자 함수형 언어인 새로운 언어를 만들었습니다.
```
[the remark this answers is not in the document]
Comment (alexkoala) [reply ko/about/index.md:14]: 'Ruby' is proper noun, so it shouldn't be translated. Currently only Korean uses localized name in website.

[the remark this answers is not in the document]
Comment (alexkoala) [reply ko/about/index.md:14]: 'Ruby' is proper noun, so it shouldn't be translated. Currently only Korean uses localized name in website.

**Comment by alexkoala** [comment]
> Seems like Korean version of Python doc site uses '파이썬' too: https://docs.python.org/ko/3/

Yes. It looks only Korean Python document uses localized name.

File: ko/about/index.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (yous) [reply ko/about/index.md]: I meant '펄, 스몰토크, 아이펠, 에이다, 리스프'.

[the remark this answers is not in the document]
Comment (alexkoala) [reply ko/about/index.md]: Sorry for misunderstanding, fixed.

File: ko/documentation/ruby-from-other-languages/index.md [hunk 7]
```diff
@@ -1,37 +1,37 @@
...
 layout: page
-title: "다른 언어에서 루비로"
+title: "다른 언어에서 Ruby로"
 lang: ko
 ---
 
-처음 루비 코드를 볼 때, 그것은 당신이 사용했던 다른 프로그래밍 언어를
-연상하게 할 수 있습니다. 이것은 의도적입니다. 대부분의 문법은 펄, 파이썬,
-자바나 다른 언어들과 유사합니다. 다른 언어를 사용한 적이 있다면 루비를
+처음  코드를 볼 때, 그것은 당신이 사용했던 다른 프로그래밍 언어를
```
Comment (yous) [inline ko/documentation/ruby-from-other-languages/index.md:7]: Yet missing some "Ruby"s. Please check this file again.

File: ko/documentation/quickstart/4/index.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: page
-title: "루비 20분 가이드"
+title: " 20분 가이드"
```
[the remark this answers is not in the document]
Comment (marocchino) [reply ko/documentation/quickstart/4/index.md:3]: Even if you're done, it doesn't seem easy to find mistakes in this amount of correction. Couldn't it be resolved more automatically?

File: ko/documentation/ruby-from-other-languages/to-ruby-from-python/index.md [hunk 9]
```diff
@@ -1,35 +1,35 @@
...
-파이썬도 매우 좋은 범용적인 프로그래밍 언어입니다. 파이썬에서 루비로
-넘어오면 파이썬을 배울 때보다 조금 더 구문이 늘었을 뿐이라는 걸
+Python도 매우 좋은 범용적인 프로그래밍 언어입니다. Python에서 Ruby로
+넘어오면 Python을 배울 때보다 조금 더 구문이 늘었을 뿐이라는 걸
 알게 될 것입니다.
 
 ### 비슷한 점들
 
-파이썬처럼, 루비에서도...
+Python처럼, 에서도...
```
Comment (yous) [inline ko/documentation/ruby-from-other-languages/to-ruby-from-python/index.md:13]: Same for this file.

File: ko/documentation/success-stories/index.md [hunk 10]
```diff
@@ -4,51 +4,51 @@ title: "사용 사례"
 lang: ko
 ---
 
-많은 사람들이 현업이나 취미로 일상적으로 루비를 사용하고 있습니다.
-여기에 참고로 실생활에서 루비를 사용하고 있는 예를 모아 보았습니다.
+많은 사람들이 현업이나 취미로 일상적으로 를 사용하고 있습니다.
+여기에 참고로 실생활에서 를 사용하고 있는 예를 모아 보았습니다.
```
Comment (yous) [inline ko/documentation/success-stories/index.md:8]: Same for this file.

File: ko/community/podcasts/index.md [hunk 11]
```diff
@@ -4,22 +4,22 @@ title: "팟캐스트"
 lang: ko
 ---
 
-루비와 루비 커뮤니티에 대한 뉴스, 인터뷰, 토론을 들어보세요.
+Ruby와 Ruby 커뮤니티에 대한 뉴스, 인터뷰, 토론을 들어보세요.
 
-[루비 로그(Ruby Rogues)][rogues]
-: 루비 로그 팟캐스트는 프로그래밍, 업계, 커뮤니티, 루비와 관련된 주제로 진행되는
+[Ruby 로그(Ruby Rogues)][rogues]
```
Comment (marocchino) [inline ko/community/podcasts/index.md:9]:
```suggestion
[Ruby Rogues][rogues]
```

File: ko/about/index.md [hunk 12]
```diff
@@ -1,62 +1,61 @@
...
+Ruby가 왜 인기 있을까요? Ruby의 팬들은 아름답고 기교가 뛰어난 언어이기 때문이라고 말합니다. 하지만 이 말로는 부족하지요. 어떤
+점이 Ruby를 이처럼 매력적인 언어로 만드는 것일까요?
 {: .summary}
 
-### 루비를 만든 마츠의 목표
+### Ruby를 만든 마츠의 목표
 
-루비는 균형잡힌 언어입니다. 루비를 만든 [유키히로 마츠모토][matz](일명 마츠)는 그가 좋아하는 언어인 펄, 스몰토크, 아이펠,
-에이다, 리스프를 섞어서 명령형 프로그래밍언어이자 함수형 언어인 새로운 언어를 만들었습니다.
+Ruby는 균형잡힌 언어입니다. Ruby를 만든 [유키히로 마츠모토][matz](일명 마츠)는 그가 좋아하는 언어인 Perl, Smalltalk, Eiffel, Ada, 그리고 Lisp를 섞어서 명령형 프로그래밍 언어이자 함수형 언어인 새로운 언어를 만들었습니다.
```
Comment (yous) [inline ko/about/index.md:13]: Maybe you can wrap this line additionally.
```suggestion
Ruby는 균형잡힌 언어입니다. Ruby를 만든 [유키히로 마츠모토][matz](일명 마츠)는 그가 좋아하는 언어인 Perl, Smalltalk, Eiffel, Ada, Lisp를 섞어서 명령형 프로그래밍 언어이자 함수형 언어인 새로운 언어를 만들었습니다.
```

File: ko/documentation/index.md [hunk 13]
```diff
@@ -65,37 +65,37 @@ lang: ko
...
-[루비 C API 레퍼런스][extensions] (영문)
-: 루비의 공식 C API 문서입니다.
-  루비 개발에 참여하거나, C 확장을 개발할 때 도움이 될 것입니다.
+[Ruby C API 레퍼런스][extensions] (영문)
+: Ruby의 공식 C API 문서입니다.
+  Ruby 개발에 참여하거나, C 확장을 개발할 때 도움이 될 것입니다.
 
 [RubyDoc.info][16] (영문)
-: 루비 젬과 GitHub에서 호스팅 되는 루비 프로젝트의 레퍼런스 문서들을
+: Ruby gem과 GitHub에서 호스팅 되는 Ruby 프로젝트의 레퍼런스 문서들을
```
Comment (marocchino) [inline ko/documentation/index.md:83]:
```suggestion
: RubyGem과 GitHub에서 호스팅 되는 Ruby 프로젝트의 레퍼런스 문서들을
```

File: ko/documentation/quickstart/4/index.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (yous) [reply ko/documentation/quickstart/4/index.md:3]: Maybe one commit for `s/루비/Ruby/g`, another commit for miscellaneous fixes?

File: ko/community/index.md [hunk 14]
```diff
@@ -4,51 +4,51 @@ title: "커뮤니티"
...
+Ruby의 장점, 특징에 대한 설명에서 빠지지 않고 등장하는 것이 바로 친절한 커뮤니티입니다. Ruby 커뮤니티는 모든 사람에게
 열려있습니다.
 {: .summary}
 
 관심이 있다면 꼭 참여해보세요. 여기 참여할 수 있는 몇 가지 방법을 알려드립니다.
 
 
-[루비 사용자 포럼(RubyKR)](http://groups.google.com/group/rubykr)
-: 한국 루비 커뮤니티입니다.
+[Ruby 사용자 포럼(RubyKR)](http://groups.google.com/group/rubykr)
```
Comment (marocchino) [inline ko/community/index.md:14]: To match the site header you should write:
```suggestion
[한국 루비 사용자 모임(RubyKR)](http://groups.google.com/group/rubykr)
```

File: ko/documentation/quickstart/4/index.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (alexkoala) [reply ko/documentation/quickstart/4/index.md:3]: Currently checking all files, sorry about too many error.

**Comment by alexkoala** [comment]
Sorry for too many error on this commit, currently fixing.

File: ko/index.html [hunk 15]
```diff
@@ -1,28 +1,40 @@
 ---
 layout: homepage
-title: "루비 : 개발자의 단짝 친구"
-lang: ko
+title: Ruby 프로그래밍 언어
+lang: en
```
[the remark this answers is not in the document]
Comment (alexkoala) [reply ko/index.html:4]: Reverted `lang`.
Should I also revert `title`? It looks like original text is `Ruby Programming Language`.

[the remark this answers is not in the document]
Comment (yous) [reply ko/index.html]: I think the title is okay.

File: ko/documentation/index.md [hunk 16]
```diff
@@ -48,15 +48,15 @@ lang: ko
...
 
-[루비 FAQ][ko-2]
+[루비 프로그래밍 언어 FAQ][ko-2]
 : 황대산 님이 정리하신 FAQ입니다.
 
 ### 매뉴얼
 
-[프로그래밍 루비][9] (영문)
-: 영어로 된 독창적인 루비 책입니다. [Pragmatic Programmers의 책][10]
+[프로그래밍 Ruby][9] (영문)
```
Comment (marocchino) [inline ko/documentation/index.md:58]: I don't know why this needs to be translated from the start.
```suggestion
[Programming Ruby][9] (영문)
```

**Review by marocchino — APPROVED** [review]
Thanks for your hard works. 감사합니다. ❤️

File: ko/about/index.md [hunk 17]
```diff
@@ -1,62 +1,61 @@
...
-### 루비의 성장세
+### Ruby의 성장세
 
-1995년에 처음 공개된 이후로 전세계에 많은 루비 개발자가 생겨났습니다. 특히 2006년에는 수많은 사람들이 루비를 배웠습니다.
-전 세계의 웬만한 도시에는 활발한 사용자 그룹이 만들어졌고, 루비와 관련된 콘퍼런스는 모두 매진되었습니다.
+1995년에 처음 공개된 이후로 전세계에 많은 Ruby 개발자가 생겨났습니다. 특히 2006년에는 수많은 사람들이 Ruby를 배웠습니다.
+전 세계의 웬만한 도시에는 활발한 사용자 그룹이 만들어졌고, Ruby와 관련된 콘퍼런스는 모두 매진되었습니다.
 
-루비의 주된 [메일링 리스트](/ko/community/mailing-lists/)인 루비 토크는 2006년까지 하루에 평균 200개의 글이 올라올 정도로
+Ruby의 주된 [메일링 리스트](/ko/community/mailing-lists/)인 Ruby 토크는 2006년까지 하루에 평균 200개의 글이 올라올 정도로
```
Comment (yous) [inline ko/about/index.md:27]:
```suggestion
Ruby의 주된 [메일링 리스트](/ko/community/mailing-lists/)인 Ruby-Talk는 2006년까지 하루에 평균 200개의 글이 올라올 정도로
```

File: ko/about/index.md [hunk 18]
```diff
@@ -1,62 +1,61 @@
...
+이런 성장은 Ruby로 만들어진 대중적인 소프트웨어, 특히 [Ruby on Rails][ror] 웹 프레임워크의 공입니다.
 
-루비는 [완전히 무료]({{ site.license.url }})입니다. 사용뿐만 아니라 복사, 수정, 배포까지도 무료입니다.
+Ruby는 [완전히 무료]({{ site.license.url }})입니다. 사용뿐만 아니라 복사, 수정, 배포까지도 무료입니다.
 
 ### 모든 것은 객체
 
-처음에 마츠는 이상적인 문법을 가진 언어를 찾고 있었습니다. 그 때를 회고하며 그는 “나는 펄보다 강력하고 파이썬보다는 객체지향적인
+처음에 마츠는 이상적인 문법을 가진 언어를 찾고 있었습니다. 그 때를 회고하며 그는 “나는 Perl보다 강력하고 Python보다는 객체지향적인
 스크립트 언어가 필요했다<sup>[2](#fn2)</sup>.”라고 했습니다.
```
Comment (yous) [inline ko/about/index.md:39]:
```suggestion
스크립트 언어가 필요했다<sup>[2](#fn2)</sup>.”고 했습니다.
```

File: ko/about/index.md [hunk 19]
```diff
@@ -121,46 +120,46 @@ end
...
+**MRI**(“Matz의 Ruby 인터프리터”)나 **CRuby**(C로 작성된 Ruby) 같은 레퍼런스 구현체(reference
+implementation)에 대해 이야기하고 있습니다만, 다른 것들도 있습니다.
 이들은 특정 상황에서 유용할 때가 있는데, 다른 언어 혹은 환경과의 추가 통합을 지원한다거나
 MRI가 지원하지 않는 특별한 기능을 가지거나 합니다.
 
-여기 목록이 있습니다:
+다음은 해당 구현체 목록입니다.
 
-* [JRuby][jruby]는 JVM(자바 가상 기기) 위에서 구동되는 루비입니다. JVM의 최적화 JIT 컴파일러, 가비지 컬렉터, 병렬 스레드, 툴 에코시스템,
+* [JRuby][jruby]는 JVM(Java 가상 기기) 위에서 구동되는 Ruby입니다. JVM의 최적화 JIT 컴파일러, 가비지 컬렉터, 병렬 스레드, 툴 에코시스템,
```
Comment (yous) [inline ko/about/index.md:150]: I think '가상 머신' is widely used.
```suggestion
* [JRuby][jruby]는 JVM(Java 가상 머신) 위에서 구동되는 Ruby입니다. JVM의 최적화 JIT 컴파일러, 가비지 컬렉터, 병렬 스레드, 툴 에코시스템,
```

File: ko/about/index.md [hunk 20]
```diff
@@ -121,46 +120,46 @@ end
...
   그리고 다양한 라이브러리 집합을 활용합니다.
-* [Rubinius][rubinius]는 ‘루비로 쓰여진 루비’입니다. LLVM 위에 구축되어,
+* [Rubinius][rubinius]는 ‘Ruby로 쓰인 Ruby’입니다. LLVM 위에 구축되어,
   Rubinius는 다른 언어 역시 구축된 멋진 가상 기기 위에서 활동합니다.
```
Comment (yous) [inline ko/about/index.md:153]:
```suggestion
  Rubinius는 다른 언어 역시 구축된 멋진 가상 머신 위에서 활동합니다.
```

File: ko/community/ruby-core/index.md [hunk 21]
```diff
@@ -39,7 +39,7 @@ $ git clone https://github.com/ruby/ruby.git
 $ git clone git@git.ruby-lang.org:ruby.git
 {% endhighlight %}
 
-### 패치들로 루비 개선하기
+### 패치들로 Ruby 개선하기
 {: #patching-ruby}
 
 코어 팀은 패치나 버그 리포트를 [이슈 트랙커][10]에서
```
Comment (yous) [inline ko/community/ruby-core/index.md:45]:
```suggestion
코어 팀은 패치나 버그 리포트를 [이슈 트래커][10]에서
```

File: ko/community/ruby-core/index.md [hunk 22]
```diff
@@ -81,21 +81,21 @@ $ git clone git@git.ruby-lang.org:ruby.git
 **주의사항:** 패치는 [unified diff][12]로 등록되어야 합니다. 패치의 머지에 관한
```
Comment (yous) [inline ko/community/ruby-core/index.md:81]: There is another '트랙커' above this line.

File: ko/community/ruby-core/index.md [hunk 23]
```diff
@@ -81,21 +81,21 @@ $ git clone git@git.ruby-lang.org:ruby.git
...
 영어와 동시에 진행되는 일어 개발 메일링 리스트도 있습니다. 클레임이 해결되지
 않았을 경우에도 인내심을 가지고 며칠 후에 다시 시도해주시길 바랍니다.
 
 ### 브랜치에 대한 안내
 {: #branches-ruby}
 
-루비의 소스 코드는 2019년 4월 22일까지 서브버전에서 관리되어 왔습니다.
+Ruby의 소스 코드는 2019년 4월 22일까지 서브버전에서 관리되어 왔습니다.
 그러므로 몇몇 버전은 그대로 서브버전에서 관리됩니다.
 다음 SVN 저장소를 이용할 수 있습니다.
```
Comment (yous) [inline ko/community/ruby-core/index.md:101]: There is another '트랙커' below this line.

File: ko/documentation/installation/index.md [hunk 24]
```diff
@@ -254,16 +254,16 @@ Windows에서는 [RubyInstaller][rubyinstaller]는 전체 루비 개발 환경
...
+* [Bitnami Ruby Stack][rubystack]는 Ruby on Rails를 위한 완전한 개발환경을 제공합니다.
   macOS, Linux, Windows, 가상 머신, 클라우드 이미지를 지원합니다.
 
 
 ## 관리자
 
-많은 루비스트들이 여러 버전의 루비를 관리하기 위해 루비 관리자를 사용합니다.
-그들은 각 프로젝트에서 서로 다른 루비 버전을 사용할 수 있도록 간단하게 또는
+많은 루비스트들이 여러 버전의 Ruby를 관리하기 위해 Ruby 관리자를 사용합니다.
+그들은 각 프로젝트에서 서로 다른 Ruby 버전을 사용할 수 있도록 간단하게 또는
```
Comment (yous) [inline ko/documentation/installation/index.md:266]:
```suggestion
각 프로젝트에서 서로 다른 Ruby 버전을 사용할 수 있도록 간단하게 또는
```

File: ko/documentation/installation/index.md [hunk 25]
```diff
@@ -272,46 +272,46 @@ Windows에서는 [RubyInstaller][rubyinstaller]는 전체 루비 개발 환경
...
 플러그인이 필요합니다(이는 [ruby-build](#ruby-build)를 사용합니다).
 
 
 ### chruby
 {: #chruby}
 
-[chruby][chruby]는 여러 루비들 간의 전환을 하게 합니다.
-chruby는 [ruby-install](#ruby-install)로 설치된 루비뿐만
-아니라 소스에서 빌드 된 루비도 관리할 수 있습니다.
+[chruby][chruby]는 여러 Ruby들 간의 전환을 하게 합니다.
```
Comment (yous) [inline ko/documentation/installation/index.md:282]:
```suggestion
[chruby][chruby]는 여러 Ruby 간의 전환을 하게 합니다.
```

File: ko/documentation/installation/index.md [hunk 26]
```diff
@@ -272,46 +272,46 @@ Windows에서는 [RubyInstaller][rubyinstaller]는 전체 루비 개발 환경
...
+유명한 플러그인에서 Ruby를 설치할 수 있습니다.
 rbenv, ruby-build 모두 macOS, Linux나 다른 UNIX-계열 운영체제에서 사용가능합니다.
 
 
 ### RVM ("Ruby Version Manager")
 {: #rvm}
 
-[RVM][rvm]은 시스템에 여러 종류의 루비를 설치하고 관리할 수 있게 해줍니다.
+[RVM][rvm]은 시스템에 여러 종류의 Ruby를 설치하고 관리할 수 있게 해줍니다.
 RVM은 다른 gemsets도 관리해 줍니다.
```
Comment (yous) [inline ko/documentation/installation/index.md:300]:
```suggestion
RVM은 다른 gemset도 관리해 줍니다.
```

File: ko/documentation/quickstart/2/index.md [hunk 27]
```diff
@@ -26,12 +26,12 @@ irb(main):012:1> end
...
-`def h`는 메서드의 정의의 시작입니다. 루비에게 `h`란 이름의 메서드의 정의를
+`def h`는 메서드의 정의의 시작입니다. Ruby에게 `h`란 이름의 메서드의 정의를
 시작한다는 것을 알리는 것이죠. 그 다음 줄은 메서드 몸통(body)입니다. 전에 본 것과
 같은 `puts "Hello World"`입니다.
-마지막으로 `end`는 루비에게 메서드 정의가 끝났음을 알려줍니다.
-`=> :h`은 메서드의 정의가 끝났음을 루비가 알아들었다는 것을 의미하죠.
-루비 2.0이나 그 이전 버전이라면 이 반환 값은 `=> nil`이 될 수도 있습니다.
+마지막으로 `end`는 Ruby에게 메서드 정의가 끝났음을 알려줍니다.
+`=> :h`은 메서드의 정의가 끝났음을 Ruby가 알아들었다는 것을 의미하죠.
+ 2.0이나 그 이전 버전이라면 이 반환 값은 `=> nil`이 될 수도 있습니다.
```
Comment (yous) [inline ko/documentation/quickstart/2/index.md:34]:
```suggestion
Ruby 2.0이나 그 이전 버전이라면 이 반환 값은 `=> nil`이 될 수도 있습니다.
```

File: ko/documentation/quickstart/4/index.md [hunk 28]
```diff
@@ -124,8 +124,8 @@ if __FILE__ == $0
 
 ## 이제 기초는 다 익혔습니다
 
-루비를 잠깐 둘러 보았습니다. 물론 훨씬 더 많은 것들이 기다리고 있습니다. 루비가 제공하는 여러 제어문들, 블록의 사용과
-“yield” 메서드, 모듈과 믹스인, 그리고 훨씬 더 많은 것들이요. 이 짧은 튜토리얼이 여러분이 루비를 더 배워보고 싶게
+Ruby를 잠깐 둘러 보았습니다. 물론 훨씬 더 많은 것들이 기다리고 있습니다. 가 제공하는 여러 제어문들, 블록의 사용과
```
Comment (yous) [inline ko/documentation/quickstart/4/index.md:127]:
```suggestion
Ruby를 잠깐 둘러 보았습니다. 물론 훨씬 더 많은 것들이 기다리고 있습니다. Ruby가 제공하는 여러 제어문들, 블록의 사용과
```

File: ko/documentation/quickstart/index.md [hunk 29]
```diff
@@ -13,29 +13,29 @@ header: |
...
+  <h1>Ruby 20분 가이드</h1>
 
 ---
 
 ## 소개
 
-이 문서는 20분 정도면 따라 해볼 수 있는 간단한 루비 튜토리얼입니다.
-이 튜토리얼을 따라 하기 위해서는 컴퓨터에 루비가 설치되어 있어야
-합니다. (아직 루비가 설치되어 있지 않다면, 시작하기 전에
+이 문서는 20분 정도면 따라 해볼 수 있는 간단한  튜토리얼입니다.
```
Comment (yous) [inline ko/documentation/quickstart/index.md:22]:
```suggestion
이 문서는 20분 정도면 따라 해볼 수 있는 간단한 Ruby 튜토리얼입니다.
```

File: ko/documentation/quickstart/index.md [hunk 30]
```diff
@@ -13,29 +13,29 @@ header: |
...
+이 튜토리얼을 따라 하기 위해서는 컴퓨터에 Ruby가 설치되어 있어야
+합니다. (아직 Ruby가 설치되어 있지 않다면, 시작하기 전에
 [설치][installation]하시기 바랍니다.)
 
-## 대화형 루비 셸(irb)
+## 대화형  셸(irb)
```
Comment (yous) [inline ko/documentation/quickstart/index.md:27]:
```suggestion
## 대화형 Ruby 셸(irb)
```

File: ko/documentation/quickstart/index.md [hunk 31]
```diff
@@ -13,29 +13,29 @@ header: |
...
 
-루비에는 당신이 입력한 루비 구문을 바로 결과로 보여주는 프로그램이
-동봉되어 있습니다. 대화형 세션에서 루비 코드를 가지고 노는 것은
-루비를 배울 수 있는 훌륭한 방법입니다.
+Ruby에는 당신이 입력한  구문을 바로 결과로 보여주는 프로그램이
+동봉되어 있습니다. 대화형 세션에서  코드를 가지고 노는 것은
+를 배울 수 있는 훌륭한 방법입니다.
```
Comment (yous) [inline ko/documentation/quickstart/index.md:31]:
```suggestion
Ruby에는 당신이 입력한 Ruby 구문을 바로 결과로 보여주는 프로그램이
동봉되어 있습니다. 대화형 세션에서 Ruby 코드를 가지고 노는 것은
Ruby를 배울 수 있는 훌륭한 방법입니다.
```

File: ko/documentation/ruby-from-other-languages/index.md [hunk 32]
```diff
@@ -1,37 +1,37 @@
...
 
-## 차이점들: *어떤 언어*에서 루비로
+## 차이점들: *어떤 언어*에서 Ruby로
 
-* [C 나 C++에서 루비로](/en/documentation/ruby-from-other-languages/to-ruby-from-c-and-cpp/)
-* [자바에서 루비로](to-ruby-from-java/)
-* [펄에서 루비로](/en/documentation/ruby-from-other-languages/to-ruby-from-perl/)
-* [PHP에서 루비로](/en/documentation/ruby-from-other-languages/to-ruby-from-php/)
-* [파이썬에서 루비로](to-ruby-from-python/)
+* [C 나 C++에서 Ruby로](/en/documentation/ruby-from-other-languages/to-ruby-from-c-and-cpp/)
```
Comment (yous) [inline ko/documentation/ruby-from-other-languages/index.md:19]:
```suggestion
* [C나 C++에서 Ruby로](/en/documentation/ruby-from-other-languages/to-ruby-from-c-and-cpp/)
```

File: ko/community/ruby-core/index.md (hunk 23 above)
Comment (alexkoala) [reply ko/community/ruby-core/index.md:101]: fixed

File: ko/documentation/installation/index.md (hunk 24 above)
Comment (alexkoala) [reply ko/documentation/installation/index.md]: fixed

**Review by yous — APPROVED** [review]

**Pull request merged**